apps/hpc-ftsadmin/src/app/utils/parse-filters.ts

```diff
@@ -99,7 +108,7 @@ const filterValueIsDayJS = (value: FilterValues): value is Dayjs => {
 };
 
 const filterValueIsFlowStatusType = (
-  value: FilterValues | number
+  value: FilterValue | number
```

Comment:
You pass argument of type `string | number` in the only usage, so you can be more specific, especially since the body of the function only works with strings
```suggestion
  value: string | number
```

Reply:
I think and for the other occurrences I will just type value as `unknown`. I feel type guards are best written and done when you expect anything and you check properly. Also its easier to use everywhere. When I was writing this I didn't have such strong beliefs :P